lib/Fez/Util/Config.rakumod

```diff
@@ -2,43 +2,27 @@ unit module Fez::Util::Config;
 
 use Fez::Util::Json;
 
-state $FALLBACK-CONFIG-PATH = (%*ENV<FEZ_FALLBACK_CONFIG>//%?RESOURCES<config.json>).IO;
-state %FALLBACK-CONFIG  = from-j($FALLBACK-CONFIG-PATH.slurp);
+state $ENV-CONFIG-PATH = (%*ENV<FEZ_CONFIG>//%?RESOURCES<config.json>).IO;
```

Comment:
Won't this stop the fallback mechanism should one specify a config path via the env variable? Is that intended?